concourse/pipelines/deploy.yml

```diff
@@ -196,6 +182,25 @@ resources:
       versioned_file: router.json
       initial_version: "0"
 
+
+  - name: frontend-image
+    type: registry-image
+    icon: docker
+    source:
+      repository: govuk/frontend
+      tag: deployed-to-production
```

Comment:
I think I'd prefer `release` to `deployed-to-production` here. I hear what you're saying about incomplete testing, but I'm pretty sure the unit tests will have run before an image with the `release` tag gets pushed. I'd like us to keep thinking about this as the left-hand-side of the pipeline and having things depend on a manual deploy to the old production environment just breaks that for me.

Reply:
cool, I can do that.